layers/memcached/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION as ext
+
+ENV LIBMEMCACHED_BUILD_DIR=${BUILD_DIR}/libmemcached
+
+RUN set -xe; \
+    mkdir -p ${LIBMEMCACHED_BUILD_DIR}; \
+# Download and upack the source code
+    curl -Ls  https://launchpad.net/libmemcached/1.0/1.0.18/+download/libmemcached-1.0.18.tar.gz  \
+  | tar xzC ${LIBMEMCACHED_BUILD_DIR}
+
+# patch from https://src.fedoraproject.org/rpms/libmemcached/raw/HEAD/f/libmemcached-build.patch for build with GCC7
+COPY libmemcached-build.patch ${LIBMEMCACHED_BUILD_DIR}/libmemcached-1.0.18/libmemcached-build.patch
+
+# Move into the unpackaged code directory
+WORKDIR  ${LIBMEMCACHED_BUILD_DIR}/libmemcached-1.0.18/
+
+# Install
+RUN set -xe; \
+    patch -d . -p0 < libmemcached-build.patch && \
+    ./configure --prefix=${INSTALL_DIR} && \
+    make && \
+    make install
+
+RUN pecl install memcache
```

Comment:
Are you sure? 
Dont you mean memcached?

Reply:
Or we maybe should add both?

Reply:
Haha funny. Yeah good question. memcache is the extension for memcached.

memcached for use with libmemcached

Reply:
I add both PHP extensions with this docker image.. Im currently building and testing.